src/QA/QA.Web/Views/Dashboard/Index.cshtml

```diff
@@ -119,7 +119,8 @@
                     <td class="govuk-table__cell">
                         <span asp-show="@vacancy.ShowAssignmentInfoCaption">@assignmentInfoCaption<br /></span>
                         <a asp-show="@vacancy.CanShowReviewLink" asp-route="@RouteNames.Vacancy_Review_Get" asp-route-reviewId="@vacancy.ReviewId" class="govuk-link">Review</a>
-                        <span asp-show="@vacancy.IsClosed">@vacancy.ClosedReviewAssignmentCaption</span>
+                        <p asp-show="@vacancy.IsVacancyDeleted">This vacancy has been deleted.</p>
+                        <span asp-show="@vacancy.IsClosed && !@vacancy.IsVacancyDeleted">@vacancy.ClosedReviewAssignmentCaption</span>
```

Comment:
We put helper properties in the VM to prevent these kind of expressions from complicating the view.  Do we care that the vacancy is not deleted?  I think it might be useful to know who reviewed the vacancy before it got deleted.  As it stands at the moment you can't delete a closed vacancy anyway so you'll never have a deleted closed vacancy so this check here is superfluous.  If this is indeed the logic can you put it into a helper bool property in the VM.

Reply:
I did speak to Jithesh and Sultan about it. They are very clear and want the message 'This vacancy is deleted' only. They do not want the text that shows who reviewed it.